tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/DistributionTests.java

```diff
@@ -108,10 +108,10 @@ public void testQuickStartGenerationAndRun() throws Exception
 
         try (JettyHomeTester.Run run1 = distribution.start(args1))
         {
-            assertTrue(run1.awaitFor(5, TimeUnit.SECONDS));
+            assertTrue(run1.awaitFor(10, TimeUnit.SECONDS));
             assertEquals(0, run1.getExitValue());
 
-            File war = distribution.resolveArtifact("org.eclipse.jetty.tests:test-simple-webapp:war:" + jettyVersion);
+            File war = distribution.resolveArtifact("org.eclipse.jetty.demos:demo-simple-jsp-webapp:war:" + jettyVersion);
```

Comment:
Should this be demo-simple-webapp or the demo-jsp-webapp? There's no demo-simple-jsp-webapp AFAIK.